src/components/timeline/GlobalTrack.js

```diff
@@ -274,6 +285,17 @@ class GlobalTrackComponent extends PureComponent<Props> {
                 </div>
               ) : null}
             </button>
+            <Localized
+              id="TrackNameButton--hide-process"
+              attrs={{ title: true }}
+            >
+              <button
+                type="button"
+                className="timelineTrackCloseButton"
+                title="Hide process"
+                onClick={this._hideCurrentTrack}
+              />
+            </Localized>
```

Comment:
I'm thinking if we should hide this button or not when there is only one hideable track left. `hideGlobalTrack` action already handles this case and doesn't remove the last track but it could be good to also hide this button for this case so people don't see this option at all.
But if we decide to do it, we can still do it as a follow-up. I'm okay with landing it as is.

Reply:
I think it's a good idea! We should also apply it to the context menu item, which currently is not disabled in that case, but does nothing.
<img width="300" alt="Capture d’écran 2022-08-10 à 11 44 30" src="https://user-images.githubusercontent.com/2633774/183870850-7ffcccdd-86fc-402f-a480-9555436855ea.png">

Reply:
Yeah, we should also fix this one.